egs/xbmu_amdo31/ASR/RESULTS.md

```diff
@@ -0,0 +1,41 @@
+## Results
+
+### XBMU-AMDO31 BPE training result (Stateless Transducer)
+
+#### Pruned transducer stateless 5
+
+[./pruned_transducer_stateless5](./pruned_transducer_stateless5)
+
+It uses pruned RNN-T.
+
+|                        | test | dev  | comment                               |
+|------------------------|------|------|---------------------------------------|
+| greedy search          | 10.99 | | --epoch 42 --avg 39 --max-duration 600 |
+
+
+Training command is:
+
+```bash
+cd egs/xbmu_amdo31/ASR
+./prepare.sh
+
+export CUDA_VISIBLE_DEVICES="0"
+./pruned_transducer_stateless5/train.py 
+```
+
+**Caution**: It uses `--context-size=1`.
+
```

Comment:
Could you also upload the pre-trained model to hugging face?

Reply:
> Could you also upload the pre-trained modelok to hugging face?
ok